main/FatesInterfaceMod.F90

```diff
@@ -808,19 +813,27 @@ subroutine SetFatesGlobalElements1(use_fates,surf_numpft,surf_numcft,param_reade
             ! If we are using fixed biogeography or no-comp then we
             ! can also apply those constraints to maxpatch_primaryland and secondary
             ! and that value will match fates_maxPatchesPerSite
+
+            if(hlm_use_luh==ifalse) then
+               maxpatches_by_landuse(secondaryland:n_landuse_cats) = 0
+            end if
             
             if(hlm_use_nocomp==itrue) then
-
                maxpatches_by_landuse(primaryland) = max(maxpatches_by_landuse(primaryland),fates_numpft)
-               maxpatch_total = sum(maxpatches_by_landuse(:))
+               !! REVIEWERS: NEED THIS NEXT LINE?
+               !maxpatches_by_landuse(secondaryland) = max(maxpatches_by_landuse(secondaryland),fates_numpft)
```

Comment:
@ckoven and others..  Don't we need to allocate at least numpft patches for the secondary lu category (and others if nocomp is on?

Reply:
and actually, this shouldn't be a maximum here, it should not be more than fates_numpft, it should be equal to

Reply:
@rgknox there is logic in the nocomp x land use initialization that limits the number of PFTs allowed on secondary (or other LU) patches to be the number of secondary patches allowed, see here: https://github.com/NGEET/fates/blob/main/main/EDInitMod.F90#L554-L578

Reply:
I see, thanks for pointing that out.  Can you confirm that you do want to make sure maxpatches_by_landuse(primary) is greater than or equal to numpft, at this location in the code?  The logic in the above mentioned line did not seem to ignore primary.

Reply:
Discussed this offline with @ckoven, questions addressed